dashboard/cron.py

```diff
@@ -261,39 +263,54 @@ def update_with_bq_access(self):
             # query to retrieve all file access events for one course
             # There is no catch if this query fails, event_store.events needs to exist
 
-            final_bq_query = []
+            final_query = []
             for k, query_obj in settings.RESOURCE_ACCESS_CONFIG.items():
                 # concatenate the multi-line presentation of query into one single string
                 query = " ".join(query_obj['query'])
 
                 if (data_last_updated is not None):
                     # insert the start time parameter for query
-                    query += " and event_time > @data_last_updated"
+                    if query_obj['query_data_last_updated_condition']:
+                        query += f" {query_obj['query_data_last_updated_condition']} "
+                    elif is_bigquery:
```

Comment:
Need to check for is_bigquery first, since query_obj['query_data_last_updated_condition'] is optional.

Reply:
I'll change it to a `.get()` so it should be safe if present or not

Reply:
That worked!